### ISSUE TITLE
fix: required attribute is not updating the corresponding input element in Combobox to inform assistive technologies of its required state.

### ISSUE THREAD

**Issue by break-stuff** [microsoft/fast#5839]
# 🐛 Bug Report

When adding the `required` attribute to the component, the `input` element does not get updated with `required` or `aria-required="true"` attributes to inform assistive technologies that the field is required.

## 💻 Repro or Code Sample

https://codesandbox.io/s/infallible-banach-fg5zmn

## 🤔 Expected Behavior

The `input` field within the component should be updated to indicate the field is requred.

## 🌍 Your Environment

* OS & Device: [e.g. MacOS, iOS, Windows, Linux] on [iPhone 7, PC]
* Browser [e.g. Microsoft Edge, Google Chrome, Apple Safari, Mozilla FireFox]
* Version [e.g. 1.8.0]

**Comment by chrisdholt** [comment]
Updating the title to be more descriptive of the problem for users searching through issues.

**Comment by chrisdholt** [comment]
assigning to @radium-v to investigate

**Comment by janechu** [comment]
Appears to be resolved by #5890
